src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double obtainTheArea();
```

Comment:
The method name `obtainTheArea` is not conventional. Consider renaming it to simply `getArea` to follow the standard Java naming conventions.